ezcontent_api.info.yml

```diff
@@ -4,9 +4,11 @@ description: 'Improves Drupal core JSONAPI experience.'
 package: EZContent API
 core_version_requirement: ^8.8 || ^9
 dependencies:
+  - ezcontent_listing:ezcontent_listing
```

Comment:
I think this should be `ezcontent:ezcontent_listing` as per https://www.drupal.org/docs/creating-custom-modules/let-drupal-know-about-your-module-with-an-infoyml-file#s-complete-example